README.md

```diff
@@ -4,7 +4,8 @@
 
 > This is a Ruby on rails web application where the user can register his expenses and organize it by categories and know how much of his budget is expending in what.
 
-## LIVE APP 👀
+
+## LIVE APP link 👀
 [HERE!](https://protected-anchorage-45932.herokuapp.com/)
 
 
```

Comment:
- [ ] OPTIONAL
Kindly change the header title _Authors_ to _Author_, as this project is done solely. This will make your ReadMe appear more professional.